packages/apsara-ui/src/Listing/Listing.tsx

```diff
@@ -48,7 +47,6 @@ const Listing = ({
                 value={searchTerm}
                 placeholder={searchPlaceholder}
                 disabled={disabled}
-                {...extraSearchProps}
```

Comment:
we still need to pass values from `searchProps` to this component.
but let's not do it by spreading them, we can manually pass the props one by one

Reply:
`searchProps` only have three fields which we're already passing and `extraSearchProps` is nothing but an empty object so we can remove it.